src/lib/api/index.ts

```diff
@@ -71,19 +70,18 @@ export async function post(agent: BskyAgent, opts: PostOpts) {
     | AppBskyEmbedImages.Main
     | AppBskyEmbedExternal.Main
     | AppBskyEmbedRecord.Main
+    | AppBskyEmbedVideo.Main
     | AppBskyEmbedRecordWithMedia.Main
     | undefined
   let reply
-  let rt = new RichText(
-    {text: opts.rawText.trimEnd()},
-    {
-      cleanNewlines: true,
-    },
-  )
+  let rt = new RichText({text: opts.rawText.trimEnd()}, {cleanNewlines: true})
 
   opts.onStateChange?.('Processing...')
+
   await rt.detectFacets(agent)
+  // @ts-expect-error REMOVE ME AFTER API PACKAGE UPDATE
```

Comment:
Just making sure this is purely a TS error and there's nothing broken here?

Reply:
The agent types don't match. I believe it's purely a TS error